includes/class-ssp-frontend.php

```diff
@@ -411,11 +415,27 @@ public function episode_meta_details ( $episode_id = 0, $context = 'content' ) {
 			}
 		}
 
+		$meta_display .= "<p>".__( 'Podcast:', 'seriously-simple-podcasting' )." ".$podcast_display."</p>";
+
 		$itunes_url = get_option( 'ss_podcasting_itunes_url', '' );
 		if ( ! empty( $itunes_url ) ) {
-			$meta_display .= $meta_sep . '<a href="' . esc_url( $itunes_url ) . '" title="' . __( 'Leave a review', 'seriously-simple-podcasting' ) . '" class="podcast-meta-itunes">' . __( 'Leave a review', 'seriously-simple-podcasting' ) . '</a>';
+			$subscribe_display .= '<a href="' . esc_url( $itunes_url ) . '" target="_BLANK" title="' . apply_filters( 'ssp_subscribe_link_name_itunes', __( 'iTunes', 'seriously-simple-podcasting' ) ) . '" class="podcast-meta-itunes">' . apply_filters( 'ssp_subscribe_link_name_itunes', __( 'iTunes', 'seriously-simple-podcasting' ) ) . '</a>';
```

Comment:
Please keep all instances of _blank as lowercase, purely to keep uniformity in the code.

Reply:
Sure thing. Changed it to lowercase 👌